pyproject.toml

```diff
@@ -46,13 +46,13 @@ dependencies = [
     "sqlalchemy>=2.0.25,<=2.1.0",
     "ruamel.yaml>=0.17,<0.19",
     "h5py>=3.10",
-    "httpx[http2]>=0.26.0,<0.28.0",
     "tqdm>=4.66.1,<4.67.0",
     "templateflow>=23.0.0",
     "lapy>=1.0.0,<2.0.0",
     "lazy_loader==0.4",
     "importlib_metadata; python_version<'3.9'",
     "looseversion==1.3.0; python_version>='3.12'",
+    "junifer_data==1.1.0",
```

Comment:
Indeed here is confusing. The `junifer_data` package can be updated, but the data can be the same, no? I mean, datalad dataset version is the same.

So in this case, to avoid confusion, i'll have the junifer-data following semver and the dataset version tag as incremental numbers.

Reply:
`junifer_data` is the client API which adds and improves the API we use to interact with the `junifer-data` datalad dataset.

- `junifer_data` in `pyproject.toml` refers to: https://github.com/juaml/junifer-data-client
- `JUNIFER_DATA_VERSION` in `junifer/data/utils.py` refers to https://github.com/juaml/junifer-data

Reply:
perfect